.pre-commit-config.yaml

```diff
@@ -0,0 +1,10 @@
+repos:
+- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+  rev: v2.5.0
+  hooks:
+  - id: pretty-format-java
+    args: [--autofix]
+  - id: pretty-format-kotlin
```

Comment:
We don't have Kotlin code. Do we need this?

Reply:
remove it